#!/usr/bin/env python
PACKAGE = "scitos_2d_navigation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# enable/disable whole layer
gen.add("enabled", bool_t, 0, "Whether to apply this plugin or not", True)

#~ # which maps to update and publish
#~ gen.add("publish_fine_map", bool_t, 0, "Update/publish fine map", def_publish_fine_map)
#~ gen.add("publish_block_map", bool_t, 0, "Update/publish block map", def_publish_block_map)
#~ gen.add("publish_input_map", bool_t, 0, "Update/publish raw input from obstacle_layer", def_publish_input_map)
#~ 
#~ # change algorithm values
#~ gen.add("stat_low", double_t, 0, "Static_middle", def_stat_low, 0.0000001, 0.4999999)
#~ gen.add("stat_low2", double_t, 0, "Static_middle", def_stat_low2, 0.0000001, 0.9999999)
#~ gen.add("stat_high", double_t, 0, "Static_middle", def_stat_high, 0.5000001, 0.9999999)
#~ gen.add("dyn_low", double_t, 0, "Static_middle", def_dyn_low, 0.0000001, 0.4999999)
#~ gen.add("dyn_high", double_t, 0, "Static_middle", def_dyn_high, 0.5000001, 0.9999999)
#~ 
#~ # enable/disable debug print outs
#~ gen.add("debug_flag", bool_t, 0, "Print debug comments", def_debug_flag)

exit(gen.generate(PACKAGE, "scitos_2d_navigation", "overwrite_layer_params"))
